README.md

```diff
@@ -133,59 +133,106 @@ Available targets:
 
 | Name | Version |
 |------|---------|
-| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 0.13 |
```

Comment:
add `README.yaml` file with all the sections required for a new module

Reply:
@htplbc  please add README.yaml as we use in all our modules.
Don't update README.md manually, our GitHub workflows do update it from README.yaml

Reply:
@aknysh README.yaml was added already - https://github.com/cloudposse/terraform-aws-mwaa/blob/initial_module/README.yaml

Reply:
README.yaml has been updated